Docs updates - data files and analysis README

jharenza opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose/implementation Section

#### What scientific question is your analysis addressing?
Adding missing documentation to analysis README and data-files-description.md


#### What was your approach?
Added requested information to the documentation


#### What GitHub issue does your pull request address?
Closes #1431 
Closes #1427 
Closes #1428 


### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?
Please check to make sure I did not miss anything

#### Is there anything that you want to discuss further?
For #1431 , did you want the file names listed in this column as well? UPDATE - I see these in another column, and updated a few files here.
For the markdown table updates, I had used a [web tool](https://tableconvert.com/markdown-to-markdown) for updates, so some of the spacing came back as if there are more changes than previously, but this was the quickest way to do it I have found.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes


### Results

#### What types of results are included (e.g., table, figure)?
updated docs


#### What is your summary of the results?
NA


#### Reproducibility Checklist

<!-- Check all those that apply or remove this section if it is not applicable.-->

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

<!-- Please review https://github.com/AlexsLemonade/OpenPBTA-analysis#documenting-your-analysis -->

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

<!-- IF YOUR PULL REQUEST IS A DATA RELEASE, PLEASE REMOVE THE [HTML COMMENT TAG](https://html.com/tags/comment-tag/) FROM THE SECTION BELOW AND COMPLETE THE CHECKLIST-->

<!-- 
### Data Release Checklist
- [ ] Is the table in doc/data-file-descriptions.md up to date?
- [ ] Is doc/data-format.md up to date?
- [ ] Is doc/release-notes.md up to date?
- [ ] Is download-data.sh up to date?
- [ ] Was download-data.sh tested and did it complete without error?
-->

sjspielman commented on doc/data-files-description.md:
```diff
@@ -15,83 +15,84 @@ This document contains information about all data files associated with this pro
+|`pbta-tcga-snv-lancet.vep.maf.gz` | PBTA/TCGA data file | [Somatic mutation calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#somatic-mutation-calling); [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc-lancet-wf.cwl) | Somatic SNV - Lancet [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/ma ...
+|`pbta-tcga-snv-mutect2.vep.maf.gz` | PBTA data file | [Somatic mutation calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#somatic-mutation-calling); [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc-mutect2_strelka2-wf.cwl) | Somatic SNV - Mutect2 [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysis/ ...
+|`pbta-tcga-snv-strelka2.vep.maf.gz` | PBTA data file | [Somatic mutation calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#somatic-mutation-calling); [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc-mutect2_strelka2-wf.cwl) | Somatic SNV - Strelka2 [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysi ...
+|`StrexomeLite_hg38_liftover_100bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 targeted panel regions used for all variant callers, each region padded by 100 bp |
+|`StrexomeLite_Targets_CrossMap_hg38_filtered_chr_prefixed.bed` | Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 targeted DNA panel bait capture regions provided by the kit manufacturer |
+|`WGS.hg38.lancet.300bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | WGS.hg38.lancet.unpadded.bed file with each region padded by 300 bp |
+|`WGS.hg38.lancet.unpadded.bed` | Reference Regions File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 WGS regions created using UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented with PASS variant calls from Strelka2 and Mutect2 |
+|`WGS.hg38.mutect2.vardict.unpadded.bed` | Reference Regions File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 BROAD Institute interval calling list (restricted to Chr1-22,X,Y,M and non-N regions) used for Mutect2 and VarDict variant callers |
+|`WGS.hg38.strelka2.unpadded.bed` | Reference Regions File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 BROAD Institute interval calling list (restricted to Chr1-22,X,Y,M) used for Strelka2 variant caller |
+|`WGS.hg38.vardict.100bp_padded.bed` | Reference Regions File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) |`WGS.hg38.mutect2.vardict.unpadded.bed` with each region padded by 100 bp used for VarDict variant caller |
+|`WXS.hg38.100bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 WXS regions provided by the kit manufacturer used for Strelka2, Mutect2, and VarDict variant callers with each region padded by 100 bp |
+|`WXS.hg38.lancet.400bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 WXS regions provided by the kit manufacturer used for Lancet variant callers with each region padded by 400 bp |
+|`intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` |
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` |
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `tcga_6k_genes.targetIntervals.bed`.  Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-a ...
+|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file described in [TCGA-capture- ...
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA- ...
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive ...
+|`whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_designed_120.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https: ...
+|`whole_exome_agilent_designed_120.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file describe ...
+|`whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_plus_tcga_6k.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https: ...
+|`whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file describe ...
```
It seems to me that the `Origin` column for all of these should be `tcga-capture-kit-investigation`?

sjspielman replied:
> We can put something like GDC, since we got via GDC?

This would make sense I think.

> The padded files were created based on the original files..

Is there not a script that records this?

jharenza replied:
Ok, I think I addressed everything, except the padding scripts, which @migbro will add to the OpenPBTA-workflows repository since they are currently _not_ captured within the SNV workflow as I thought.

sjspielman reviewed (commented):
In general, I think we should remove links to specific manuscript sections from these docs. Those headers are not necessarily going to stay the same and it will be easy for those links to break over time. The info at the top of this document  further describes `Origin` as either a workflow or a module link, so let's keep it just to those. 

That said, we might also want to amend that top description from 
> When applicable, a link to the specific script that produced (or modified, for Modified reference file types) the data.

to...

> When applicable, a link to the specific **module** that produced (or modified, for Modified reference file types) the data.

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
-|`whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file describe ...
+| **File name** | **File Type** | **Origin** | **File Description** |
+|---|---|---|---|
+|`fusion_summary_embryonal_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of embryonal tumor fusions of interest |
```
```suggestion
|`fusion_summary_embryonal_foi.tsv` | Analysis file | [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of embryonal tumor fusions of interest |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+| **File name** | **File Type** | **Origin** | **File Description** |
+|---|---|---|---|
+|`fusion_summary_embryonal_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of embryonal tumor fusions of interest |
+|`fusion_summary_ependymoma_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of ependymal tumor fusions of interest |
```
```suggestion
|`fusion_summary_ependymoma_foi.tsv` | Analysis file | [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of ependymal tumor fusions of interest |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|---|---|---|---|
+|`fusion_summary_embryonal_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of embryonal tumor fusions of interest |
+|`fusion_summary_ependymoma_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of ependymal tumor fusions of interest |
+|`fusion_summary_ewings_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of Ewing's sarcoma fusions of interest |
```
```suggestion
|`fusion_summary_ewings_foi.tsv` | Analysis file | [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of Ewing's sarcoma fusions of interest |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`fusion_summary_ewings_foi.tsv` | Analysis file | [`analysis/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of Ewing's sarcoma fusions of interest |
+|`gencode.v27.primary_assembly.annotation.gtf.gz` | Reference file | GENCODE v27 | hg38 gene annotation on primary assembly (reference chromosomes and scaffolds) |
+|`GRCh38.primary_assembly.genome.fa.gz` | Reference Genome file | GENCODE v27 | hg38 primary assembly genome sequence FASTA file |
+|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
```
```suggestion
|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`gencode.v27.primary_assembly.annotation.gtf.gz` | Reference file | GENCODE v27 | hg38 gene annotation on primary assembly (reference chromosomes and scaffolds) |
+|`GRCh38.primary_assembly.genome.fa.gz` | Reference Genome file | GENCODE v27 | hg38 primary assembly genome sequence FASTA file |
+|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
```
```suggestion
|`independent-specimens.wgs.primary.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`GRCh38.primary_assembly.genome.fa.gz` | Reference Genome file | GENCODE v27 | hg38 primary assembly genome sequence FASTA file |
+|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
+|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
```
```suggestion
|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
+|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
```
```suggestion
|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`independent-specimens.wgs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
+|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
+|`intersect_cds_lancet.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
```
```suggestion
|`intersect_cds_lancet.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
+|`intersect_cds_lancet.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
+|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions |
```
```suggestion
|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
+|`intersect_cds_lancet.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
+|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions |
+|`intersect_strelka_mutect_WGS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Strelka2 and Mutect2 regions called |
```
```suggestion
|`intersect_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Strelka2 and Mutect2 regions called |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`pbta-gene-expression-rsem-fpkm.stranded.rds` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM FPKM for stranded samples (gene-level) |
+|`pbta-gene-expression-rsem-tpm.polya.rds` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for poly-A samples (gene-level) |
+|`pbta-gene-expression-rsem-tpm.stranded.rds` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for stranded samples (gene-level) |
+|`pbta-histologies-base.tsv` | PBTA data file |  | Harmonized clinical metadata file  |
```
```suggestion
```
Extra line

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`pbta-gene-expression-rsem-tpm.polya.rds` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for poly-A samples (gene-level) |
+|`pbta-gene-expression-rsem-tpm.stranded.rds` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for stranded samples (gene-level) |
+|`pbta-histologies-base.tsv` | PBTA data file |  | Harmonized clinical metadata file  |
+|`pbta-histologies.tsv` | Analysis file | [`molecular-subtype-integrate`] | Harmonized clinical metadata file plus biospecimen molecular subtypes|
```
```suggestion
|`pbta-histologies.tsv` | Analysis file | [`molecular-subtype-integrate`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-integrate) | Harmonized clinical metadata file plus biospecimen molecular subtypes|
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`WXS.hg38.lancet.400bp_padded.bed` | Modified Reference File | SNV and INDEL calling | hg38 WXS regions provided by the kit manufacturer used for Lancet variant callers with each region padded by 400 bp |
+|`intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` |
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` |
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Modified Reference File [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `tcga_6k_genes.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
```
```suggestion
|`tcga_6k_genes.targetIntervals.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `tcga_6k_genes.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,91 @@ This document contains information about all data files associated with this pro
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` |
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Modified Reference File [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `tcga_6k_genes.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
+|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | Reference Target/Baits File | Downloaded via [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py)  | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/User ...
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed` | Modified Reference File [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
```
```suggestion
|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
```

sjspielman reviewed (commented):
Left a couple minor changes to clean up the table, including removing `analyses/` (sometimes misspelled `analysis`) for overall consistency. There were also a few places where a table separator was missing.

To confirm - the link we're waiting to hear on is the "SNV and Indel Calling," right?

jharenza commented:
@sjspielman I think this is ready for re-review. I applied your changes and updated the BED file origins - if it was a reference file we received, linked to cavatica and if we generated, added the workflow link.

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
 + **Origin**
 	+ For _PBTA data files_, a link the relevant D3B Center or Kids First workflow (and version where applicable).
-	+ When applicable, a link to the specific *script* that produced (or modified, for *Modified reference file* types) the data.
+	+ When applicable, a link to the specific *module* that produced (or modified, for *Modified reference file* types) the data.
```
Since we also have workflows, let's just rephrase this a tad. I don't feel super strongly about this suggestion so whatever you think! 
```suggestion
	+ When applicable, a link to the specific module, workflow, or resource that produced (or modified, for *Modified reference file* types) the data.
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`intersect_cds_lancet.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
+|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions |
+|`intersect_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Strelka2 and Mutect2 regions called |
+|`pbta-cnv-cnvkit-gistic.zip` | PBTA data file | [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/bash/run-gistic.sh) | Somatic CNV - GISTIC 2.0 output using `pbta-cnv-cnvkit.seg` file input (WGS samples only) |
```
dead link! Maybe this? https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`pbta-star-log-final.tar.gz` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | STAR log final output files |
+|`pbta-star-log-manifest.tsv` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | File to map STAR output to biospecimen IDs |
+|`pbta-sv-manta.tsv.gz` | PBTA data file | Structural variant calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_strelka2_mutect2_manta_workflow.cwl) | Somatic Structural Variant - Manta output, annotated with AnnotSV (WGS samples only) |
+|`pbta-tcga-manifest.tsv` | PBTA data file | Somatic mutation calling | Manifest of tumor/normal BAMs used for SNV calling, Tumor_Sample_Barcodes, and histologies |
```
Missing a link - probably the same workflow as above/below list items?

jharenza replied:
Ahhh - this was sort of made on the fly from the CAVATICA project after running the tumor/normals. Let me see if this exists in the repo.

jharenza replied:
@sjspielman for the above, there is no associated script. so the latest commit is the best I can do here.

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` |
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` |
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `tcga_6k_genes.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
+|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | Reference Target/Baits File | Downloaded via [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py)  | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/User ...
```
extra column 
```suggestion
|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | Downloaded via [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py)  | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) |
```

sjspielman reviewed (commented):
A couple more things I missed.. tables always get me! 🙃 

I also totally glossed over that we don't have a link for GENCODE. This is such a standard that I don't think a link is strictly _needed_, but it can't hurt. I don't feel strongly about this particular one though. Up to you - https://www.gencodegenes.org/human/release_27.html

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` |
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` |
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Modified Reference File | [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/) | hg38 version of `tcga_6k_genes.targetIntervals.bed` generated using CrossMap and bedtools sort and merge |
+|`tcga_6k_genes.targetIntervals.bed` | Reference File | Reference File | Downloaded via [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py)  | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retriev ...
```
```suggestion
|`tcga_6k_genes.targetIntervals.bed` | Reference File | Downloaded via [`tcga-capture-kit-investigation`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py)  | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) |
```

sjspielman commented on doc/data-files-description.md:
```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`pbta-star-log-final.tar.gz` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | STAR log final output files |
+|`pbta-star-log-manifest.tsv` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | File to map STAR output to biospecimen IDs |
+|`pbta-sv-manta.tsv.gz` | PBTA data file | Structural variant calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_strelka2_mutect2_manta_workflow.cwl) | Somatic Structural Variant - Manta output, annotated with AnnotSV (WGS samples only) |
+|`pbta-tcga-manifest.tsv` | PBTA data file | Somatic mutation calling | Manifest of TCGA tumor/normal BAMs used for SNV calling, Tumor_Sample_Barcodes, and histologies; information retrieved from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) |
```
Let's re-arrange this a bit given that there is no originating script, and the current info in that column makes it look like a workflow link is missing - 

```suggestion
|`pbta-tcga-manifest.tsv` | PBTA data file | Retrieved from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) | Manifest of TCGA tumor/normal BAMs used for SNV calling, Tumor_Sample_Barcodes, and histologies |
```

sjspielman reviewed (approved):
I'm approving now under the assumption that the two changes I suggested will be incorporated. Re-request review if something looks funky with those!
